gui/wxpython/projpicker_gui/gui_common.py

```diff
@@ -0,0 +1,250 @@
+"""
+This module implements common functions for the gui modules.
+"""
+
+import os
+
+import grass.projpicker as ppik
+
+projpicker_coordinates_env = "PROJPICKER_COORDINATES"
```

Comment:
[PEP 8](https://www.python.org/dev/peps/pep-0008/#global-variable-names) recommends to mark globals as private or use `__all__`.

Reply:
Done in 2ca331f